spec/lib/services/sku-pack-service-spec.js

```diff
@@ -107,8 +117,22 @@ describe("SKU Pack Service", function() {
                 destroy: sinon.stub()
             };
 
+            waterline.templates = {
```

Comment:
@yyscamper This identical pattern appears throughout the unittests.  Each unittest sets up what they need at the start (as I did) and overrides what was there by doing so. I can understand your concern of someone accidentally forgetting to setup their mock and unintentionally using mine.  I will make a change to delete these in the after().